frontend/src/container/LogsExplorerList/InfinityTableView/index.tsx

```diff
@@ -77,6 +81,8 @@ const InfinityTable = forwardRef<TableVirtuosoHandle, InfinityTableProps>(
 			Record<string, unknown>
 		>(LOCALSTORAGE.LOGS_LIST_COLUMNS);
 
+		const isDarkMode = useIsDarkMode();
```

Comment:
Can we use the isDarkMode declared at component level here ?

Reply:
didn't get this one @YounixM

Reply:
component is used as a function and props are directly mapped that's reason i imported the `useDarkMode` hook twice